client/benches/tps/dev.rs

```diff
@@ -2,11 +2,6 @@
 //! using [criterion](https://github.com/bheisler/criterion.rs)
 //! for performance check during development
 #![allow(missing_docs)]
-#![allow(
-    clippy::arithmetic_side_effects,
-    clippy::std_instead_of_core,
-    clippy::std_instead_of_alloc
-)]
```

Comment:
So you just removed these lints from Cargo.toml?

Reply:
it's not me who removed them. They have been allowed long ago

Reply:
it's not me who removed them. They have been allowed long ago

Reply:
it really doesn't make sense to enable this in every crate. We'll do some lints revision in the future